repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/dataset.ftl

```diff
@@ -100,13 +100,13 @@
         <ul data-uipath="ps.dataset.resources">
             <#list dataset.files as attachment>
                 <li class="attachment">
-                    <a title="${attachment.text}" href="<@hst.link hippobean=attachment.resource/>">${attachment.text}</a>;
+                    <a title="${attachment.text}" href="<@hst.link hippobean=attachment.resource/>" onClick="logGoogleAnalyticsEvent('Download attachment','<@fmt.message key='labels.dataset'/>','${attachment.resource.filename}');">${attachment.text}</a>;
```

Comment:
Not sure we want `<@fmt.message key='labels.dataset'/>` to be the category key as this can be changed in the CMS which would mess up the categories in GA. Maybe would be better to hardcode these to static values?

Reply:
Yes, changing the value means a new "category" dimension is created in GA, and there will be a split of the data (i.e. "Data set" downloads and "Dataset" downloads.  In this case, they are still the same category so yes hardcoded value would be better. Nice spot.